src/chi/components/alert/_alert-common.scss

```diff
@@ -64,8 +74,28 @@ $states: (
 
   @each $state in map-keys($states) {
     &.-#{$state} {
-      background-color: map-get(map-get($states, $state), background-color);
-      border-color: map-get(map-get($states, $state), border-color);
+      border-left: none;
+      color: set-color(grey, 100);
+      &.-#{$state}.-bubble::before, &.-#{$state}.-toast::before {
```

Comment:
&.-#{$state}.-toast::before should be in another line for better code readability